### DISCUSSION TITLE
Exit app with click menu option

### DISCUSSION THREAD

**Discussion by serascandia** [quasarframework/quasar#13822] [CLI - Capacitor / Cordova modes]
I have an app with capacitor, and I can't achieve exit the app, basically I have a side menu with the exit option I want when the user press the exit option the app close it.  I try with this

- navigator.app.exitApp();
- window.close()
- and plugin of capacitor @capacitor/app with exitApp()

and no luck, any can give and idea how to made this works thank you.

**Comment by otabekoff** [comment]
I've also this problem already.
